src/java.desktop/share/classes/javax/swing/plaf/metal/MetalBorders.java

```diff
@@ -1108,11 +1108,11 @@ public void paintBorder(Component c, Graphics g, int x, int y, int w, int h) {
             g.translate( x, y );
 
             g.setColor( MetalLookAndFeel.getControlDarkShadow() );
-            g.drawLine( w-1, 0, w-1, h-1 );
-            g.drawLine( 1, h-1, w-1, h-1 );
+            SwingUtilities2.drawVLine(g, w-1, 0, h-1);
+            SwingUtilities2.drawHLine(g, 1, w-1, h-1);
             g.setColor( MetalLookAndFeel.getControlHighlight() );
-            g.drawLine( 0, 0, w-2, 0 );
-            g.drawLine( 0, 0, 0, h-2 );
+            SwingUtilities2.drawHLine(g, 0, w-2, 0 );
```

Comment:
```suggestion
            SwingUtilities2.drawHLine(g, 0, w-2, 0);
```
You removed other spaces but this one's left.